tox.ini

```diff
@@ -3,17 +3,19 @@ isolated_build = true
 envlist =
     docs
     packaging
-    py{38,39,310}-dj{32,41,42}-{sqlite,postgresql,postgis,mysql}
+    py{38,39,310}-dj{32,40,41}-{sqlite,legacy_postgresql,postgis,mysql}
     py{310}-dj{40}-{sqlite}
-    py{310,311}-dj{41,42,main}-{sqlite,postgresql,postgis,mysql}
+    py{310,311}-dj{40,41}-{sqlite,legacy_postgresql,postgis,mysql}
```

Comment:
Now you're adding back many Django 4.0 test runs, those have been removed on purpose in https://github.com/jazzband/django-debug-toolbar/pull/1719. That's not better.

Reply:
my bad, should be good this time

Reply:
It seems that github actions still tries to run python3.8 with django 4.2a
https://github.com/jazzband/django-debug-toolbar/actions/runs/4113633936/jobs/7102476809#step:9:26

can you help adjust the test to prevent that?